test/mason/pkgconfig-tests/zlib.good

```diff
@@ -1,15 +1,16 @@
+Skipping registry update since no dependency found in manifest file.
 
 [root]
 name = "zlib"
 version = "0.1.0"
-chplVersion = "CHPL_CUR_FULL..CHPL_CUR_FULL"
+chplVersion = "1.23.0..1.23.0"
```

Comment:
@ankingcodes  I don't think we want to change this. `CHPL_CUR_FULL` should get replaced with the version number in the prediff.